src/components/modus-wc-badge/modus-wc-badge.stories.ts

```diff
@@ -89,3 +89,35 @@ export const WithIcon: Story = {
     `;
   },
 };
+
+export const Migration: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: `
+#### Breaking Changes
+
+  - Size values changed from 'small'/'medium'/'large' to 'sm'/'md'/'lg'
+  - The 'dark' color option is now 'high-contrast'
+  - The 'type' prop is now 'variant' and 'default' type is now 'filled'
+
+#### Prop Mapping
+
+| 1.0 Prop   | 2.0 Prop     | Notes                                   |
+|------------|------------- |----------------------------------------|
+| ariaLabel  | aria-label   | Now uses inherited attributes pattern   |
```

Comment:
```suggestion
| aria-label  | aria-label   | Now uses inherited attributes pattern   |
```

Not sure it adds value to specify inherited attributes pattern, nobody knows what that is.

Reply:
fixed in updated PR